src/GraphQL.Tests/Execution/EnumAsInputsTests.cs

```diff
@@ -184,9 +204,9 @@ public GenderEnum()
         {
             Name = "Gender";
             Description = "User gender";
-            AddValue("NotSpecified", "NotSpecified gender.", Gender.NotSpecified);
-            AddValue("Male", "gender Male", Gender.Male);
-            AddValue("Female", "gender female", Gender.Female);
+            AddValue("NotSpecified", "NotSpecified gender.", (int)Gender.NotSpecified);
+            AddValue("Male", "gender Male", (int)Gender.Male);
+            AddValue("Female", "gender female", (int)Gender.Female);
```

Comment:
This one needs to be kept as it was, since other users are using it like it was (with the enumeration value vs. the base type value) and I want to make sure there are tests that cover that usage.